### PR TITLE
Update Xiaomi.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/Xiaomi.cn.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		zhen.hd.xiaomi.cn
+
+	No acess:
+		home.xiaomi.cn
+-->
+
+<ruleset name="Xiaomi.cn">
+
+	<target host="xiaomi.cn" />
+	<rule from="^http://xiaomi\.cn/" to="https://www.xiaomi.cn/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Xiaomi.cn.xml]: Please make a comment about the error.

File: src/chrome/content/rules/Xiaomi.cn.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		mipop.xiaomi.cn (test : http://mipop.xiaomi.cn/default/activity/key/20151114 )
+		rom.xiaomi.cn
+		static.xiaomi.cn
+
+	Invalid certificate:
+		kong.bbs.xiaomi.cn
+		buglist.xiaomi.cn
+		mi3.hd.xiaomi.cn
+		michina.hd.xiaomi.cn
+		zhen.hd.xiaomi.cn
```
Comment (J0WI) [inline src/chrome/content/rules/Xiaomi.cn.xml]: error.www.xiaomi.cn

File: src/chrome/content/rules/Mi-img.com.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+<ruleset name="Mi-img.com">
+	<target host="cdn.cnbj1.fds.api.mi-img.com" />
+		<test url="http://cdn.cnbj1.fds.api.mi-img.com/src/gift/17f2d18a3442813daa3f26ce2fad47.jpg" />
+	<target host="cdn.cnbj2.fds.api.mi-img.com" />
+	<target host="ssl-cdn.static.browser.mi-img.com" />
+		<test url="http://ssl-cdn.static.browser.mi-img.com/hao/y3Y4qgA6.png" />
+
+	<rule from="^http:" to="https:" />
+</ruleset>
```
Comment (J0WI) [inline src/chrome/content/rules/Mi-img.com.xml]: See also:
s1.mi-img.com
s2.mi-img.com
cdn.awsbj0.fds.api.mi-img.com
cdn.hao.mi-img.com

File: src/chrome/content/rules/Mi.com.xml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+		ljm.mi.com
+		mahjong.mi.com
+		mobile.mi.com
+		pong.mi.com
+		popo.mi.com
+		qiye.mi.com
+		tuan.qiye.mi.com
+		texas.mi.com
+		theme.mi.com
+		tv.mi.com
```
Comment (J0WI) [inline src/chrome/content/rules/Mi.com.xml]: www.mi.com

File: src/chrome/content/rules/Mi.com.xml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+	<target host="s1.mi.com" />
+		<test url="http://s1.mi.com/js/xmst.js" />
+	<target host="static.mi.com" />
+		<test url="http://static.mi.com/cart/" />
+	<target host="dl.zb.mi.com" />
+		<test url="http://dl.zb.mi.com/3026185@style@480jpg" />
+	<target host="zbtupian.zb.mi.com" />
+		<test url="http://zbtupian.zb.mi.com/pic_85ae03f432b6d02ad49478fe22d0ed86_1478249515349.jpg" />
+
+	<rule from="^http:" to="https:" />
```
Comment (J0WI) [inline src/chrome/content/rules/Mi.com.xml]: See also:
global.mi.com
store.mi.com

File: src/chrome/content/rules/Mi-img.com.xml (hunk 3 above)
Comment (ivysrono) [inline src/chrome/content/rules/Mi-img.com.xml]: cdn.hao.mi-img.com has existed.

File: src/chrome/content/rules/Mi.com.xml (hunk 5 above)
Comment (ivysrono) [inline src/chrome/content/rules/Mi.com.xml]: s.zb.mi.com service.10046.mi.com
have existed.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**